themes/openy_themes/openy_carnation/package.json

```diff
@@ -9,35 +9,48 @@
     "breakpoint-sass": "^2.7.1",
     "cryptiles": "^4.1.2",
     "diff": "^3.5.0",
-    "eslint": "^4.18.2",
     "extend": "^3.0.2",
     "fstream": "^1.0.12",
-    "grunt-cli": "^1.3.2",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-sass-globbing": "^1.5.1",
     "hoek": "^4.2.1",
+    "jquery": "^3.4.1",
     "jquery-match-height": "^0.7.2",
     "js-yaml": "^3.13.1",
     "load-grunt-tasks": "^5.0.0",
     "lodash": "^4.17.13",
     "merge": "^1.2.1",
-    "node-sass": "^4.12.0",
+    "popper.js": "^1.16.0",
     "sass": "^1.22.4",
     "tar": "^2.2.2",
     "tunnel-agent": "^0.6.0",
-    "underscore.string": "^3.3.5"
+    "underscore.string": "^3.3.5",
+    "webpack-merge": "^4.2.2"
   },
   "devDependencies": {
-    "grunt": "^1.0.4",
-    "grunt-autoprefixer": "^3.0.4",
-    "grunt-contrib-jshint": "^2.1.0",
-    "grunt-contrib-uglify": "^4.0.1",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-sass": "^2.1.0",
-    "grunt-sass-lint": "^0.2.4",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-preset-env": "^1.7.0",
+    "copy-webpack-plugin": "^5.0.4",
+    "css-loader": "^3.2.0",
+    "eslint": "^6.6.0",
+    "eslint-loader": "^3.0.2",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-sass": "^4.13.0",
```

Comment:
let's also add polyfill for somehow IE11 compatible result
see https://github.com/ymcatwincities/openy/pull/1782/files#diff-aaaa85ee255ae55783f9abea93dc0bf5R12